openassessment/xblock/config_mixin.py

```diff
@@ -12,6 +12,11 @@
 WAFFLE_NAMESPACE = 'openresponseassessment'
 
 ALL_FILES_URLS = 'all_files_urls'
+# .. toggle_name: FEATURES['ENABLE_ORA_MOBILE_SUPPORT']
```

Comment:
How would you feel about moving these annotations above line 129? If not, above line 26?  It would be nice to move it closer to something that aligns with the `annotation_name`. Line 129 is closer to seeing the default as well.

We could also consider updating line 129 to:
```
return settings.FEATURES.get('ENABLE_ORA_MOBILE_SUPPORT') is True
```
And removing `MOBILE_SUPPORT` altogether. This would make it even more clear that the flag and switch aren't supported for this. It does add a little risk to what would otherwise just be documentation, so I'll leave this to you.

Reply:
Done! You can review this again!